cirrus/server/cirrus/main.py

```diff
@@ -1,24 +1,41 @@
+import logging
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler  # type: ignore
+from decouple import config  # type: ignore
 from fastapi import FastAPI, status
 
 from .experiment_recipes import RemoteSettings
 from .feature_manifest import FeatureManifestLanguage as FML
 from .manifest_loader import ManifestLoader
 from .sdk import SDK
 
+logger = logging.getLogger(__name__)
+
 app = FastAPI()
 remote_setting = RemoteSettings()
 sdk = SDK()
 manifest_loader = ManifestLoader()
 fml = FML()
 
 
+remote_setting_refresh_rate_in_seconds: int = int(
+    config("REMOTE_SETTING_REFRESH_RATE_IN_SECONDS", default=10)  # type: ignore
+)
```

Comment:
Settings are a little littered around here.  Might be a good idea to have a `settings.py` at the root that does all the `config` loading and then import from settings everywhere else rather than having them littered throughout the project?

Reply:
good point 👍